### PR TITLE
Milestone 3 - Connect Redux Actions

### PR DESCRIPTION
**Pull request by Zubenna**
## I did the following in this milestone;
- Made Bookstore component stateful.
- Added Remove book column in BooksList component.
- Implemented Add book and Removebook actions.

### CODE REVIEW

File: src/containers/BooksList.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      <th>Category</th>
+    </tr>
+    <tbody>
+      {books.map(book => (
+        <Book key={book.id} book={book} removeBook={removeBook} />
+      ))}
+    </tbody>
+  </table>
+);
+
```
Comment (addod19) [inline src/containers/BooksList.js:31]: - We were asked to implement handleRemoveBook. Let us kindly implement the handleRemoveBook function

File: src/containers/BooksList.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+const BooksList = ({ books, removeBook }) => (
+  <table>
+    <tr>
+      <th>Book ID</th>
+      <th>Title</th>
+      <th>Category</th>
+    </tr>
+    <tbody>
+      {books.map(book => (
+        <Book key={book.id} book={book} removeBook={removeBook} />
```
Comment (addod19) [inline src/containers/BooksList.js]: Also, we need to pass the handleRemoveBook function to the Book component

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @Zubenna, congratulations on this project :+1: The README and the PR look professional :+1: The code is clean :100: The BookFOrm component is implemented well :100: The Book component stays presentational and attaches the "remove book" to the click event :slightly_smiling_face: 
Also, remember to check the microverse group linter files and make sure it is the same as the one you are using. [requirements](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json)

However, before we can approve the project :+1: Let us kindly address the comments below

[Optional]
- Let us kindly add validation so a user can not save empty data, since it will be cost-effective and slow performance


Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations @Zubenna :tada: :clinking_glasses: 

_**Thank you for fixing the issues raised by the last reviewer.**_

**This project has passed all the mandatory requirements :clap: 
It is therefore approved :heavy_check_mark:**

**Comment(s)**
- Against your next milestone, please fix the broken `MIT` license link on your Readme.
- Do not forget to push your changes to deployment.
- If your project does not have test cases, kindly remove the `Testing` section of the Readme.

Again, congratulations, you can now start the next milestone.
Do not forget to fill the project completion form, as this is important for your weekly progress score.

**_Good luck with your next task._**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request merged**